images/tier2/evaluate-all.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+  <path fill="none" d="M12,2H4A2.006,2.006,0,0,0,2,4V16a2.006,2.006,0,0,0,2,2V8A4,4,0,0,1,8,4h6A2,2,0,0,0,12,2Z"/>
```

Comment:
Looks like we may need to modify our `d2l-icon` component [in core here](https://github.com/BrightspaceUI/core/blob/master/components/icons/icon.js#L142) to not remove the `fill` attribute if it's value is `none`, or perhaps, something other than `#494c4e`.

Reply:
Yeah agreed. We can do that when we pull these new icons in.